src/main/java/jerome/common/DateTimeHandler.java

```diff
@@ -0,0 +1,73 @@
+package jerome.common;
+
+import java.time.DateTimeException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+import jerome.exception.MalformedUserInputException;
+
+/**
+ * Represents a handler for date and time operations.
+ */
+public class DateTimeHandler {
+
+    // Solution adapted from:
+    // https://stackoverflow.com/questions/22463062/how-can-i-parse-format-dates-with-localdatetime-java-8
+    /**
+     * Represents the date input format string.
+     */
+    public static final String DATE_INPUT_FORMAT_STRING = "yyyy-MM-dd";
+    /**
+     * Represents the format for parsing input dates.
+     */
+    public static final DateTimeFormatter DATE_INPUT_FORMAT = DateTimeFormatter.ofPattern(DATE_INPUT_FORMAT_STRING);
+    /**
+     * Represents the format for outputting dates.
+     */
+    public static final DateTimeFormatter DATE_OUTPUT_FORMAT = DateTimeFormatter.ofPattern("MMM dd yyyy");
+
+    private LocalDate localDate;
+
+    /**
+     * Represents a handler for date and time operations.
+     * @param dateTime date in String format to be parsed.
+     * @throws MalformedUserInputException if the user input is dirty.
+     */
```

Comment:
```suggestion
    /**
     * Represents a handler for date and time operations.
     *
     * @param dateTime date in String format to be parsed.
     * @throws MalformedUserInputException if the user input is dirty.
     */
```

Can consider adding a new line here to fit the java coding standard beter

Reply:
Thank you @ChrisHo1341. I will implement your feedback!